examples/ipv6-eks-cluster/main.tf

```diff
@@ -55,11 +56,11 @@ provider "helm" {
 }
 
 locals {
-  tenant      = "aws001"  # AWS account name or unique id for tenant
-  environment = "preprod" # Environment area eg., preprod or prod
-  zone        = "dev"     # Environment with in one sub_tenant or business unit
-
-  kubernetes_version = "1.21"
+  tenant                  = "ipv6"    # AWS account name or unique id for tenant
+  environment             = "preprod" # Environment area eg., preprod or prod
+  zone                    = "dev"     # Environment with in one sub_tenant or business unit
+  count_availability_zone = (length(data.aws_availability_zones.available.names) <= 3) ? length(data.aws_availability_zones.available.zone_ids) : 3
```

Comment:
can we just clip to 3 always as with other examples?

Reply:
Good spot. We are not using that variable in the example. Will delete that